booster/library/Booster/Definition/Attributes/Base.hs

```diff
@@ -64,14 +65,15 @@ import Booster.SMT.Base (SExpr)
 import Booster.Util (Flag (..))
 import Booster.Util qualified as Util
 
-data DefinitionAttributes = DefinitionAttributes
-    {
+newtype DefinitionAttributes = DefinitionAttributes
+    { indexCells :: Maybe [ByteString] -- which cells to index for rewriting
```

Comment:
Perhaps worth making it a non-empty list, as we will always have a least one cell to index?

Reply:
True, running without any index cell is probably not a use case (though possible now).
We could model this by taking the empty list to the default `k` index.

Reply:
I implemented the interpretation of `[]` as "use the default" now.